README.md

```diff
@@ -101,12 +115,12 @@ For an example of this in Swift, please see the
 
 ## Having build problems?
 
-grpc-swift depends on Swift, Xcode, and swift-proto. We are currently
+grpc-swift depends on Swift, Xcode, and swift-protobuf. We are currently
 testing with the following versions:
 
 - Xcode 9.1
 - Swift 4.0-dev
```

Comment:
Could you change this to „Swift 4.0“ while at it?

Reply:
Updated in https://github.com/grpc/grpc-swift/pull/216